app/src/main/res/values/strings.xml

```diff
@@ -265,6 +265,11 @@
 	<!-- Summary for the preference that forces the service to stay running in the background. -->
 	<string name="pref_conn_persist_summary">"Force connections to stay connected while in background"</string>
 
+	<!-- Name for the retries size preference -->
+	<string name="pref_retries_title">"Persist retries"</string>
+	<!-- Description of the retries size preference -->
```

Comment:
A bit more context of where this string appears will help. Translators only see these comments and not the context of what is retrying, where in the UI this will be seen, or what it is used for.